src/coinbase/staking_reward.ts

```diff
@@ -100,6 +100,15 @@ export class StakingReward {
     return new Date(this.model.date);
   }
 
+  /**
+   * Returns the onchain address of the StakingReward.
+   *
+   * @returns The onchain address.
+   */
+  public addressId(): string {
```

Comment:
Is there a reason this is `address_id` instead of simply `address`?

Reply:
No specific reason. We refer to onchain addresses as address id within the SDK and cdp-service backend

Reply:
Got it. I think `address_id` might be more verbose than needed (and perhaps a bit confusing). But I won't block on this PR for a naming convention that's more widespread - we can change in the future if needed.